index.bs

```diff
@@ -11302,6 +11306,174 @@ The [=remote end steps=] given |session| and |command parameters| are:
 </div>
 
 
+## The webExtension Module ## {#module-webExtension}
```

Comment:
Hm, not exactly sure what we actually agreed on. Should the module's name be singular or plural? For other modules we only use singular, so that  all the references below would have to be updated as well.

Reply:
updated